Doctors-MSSC/src/main/java/com/clinics/doctors/ui/service/AppointmentService.java

```diff
@@ -0,0 +1,126 @@
+package com.clinics.doctors.ui.service;
+
+import com.clinics.common.DTO.request.outer.doctor.AddEditAppointmentDTO;
+import com.clinics.common.DTO.response.outer.AppointmentResponseDTO;
+import com.clinics.doctors.ui.model.Appointment;
+import com.clinics.doctors.ui.model.Calendar;
+import com.clinics.doctors.ui.repositorie.RepositoryAppointment;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Slf4j
+@Transactional
+@Service
+public class AppointmentService {
+
+	final private RepositoryAppointment appointmentRepository;
+	final private CalendarService calendarService;
+	final private DoctorService doctorService;
+	final private ModelMapper modelMapper;
+
+	public AppointmentService(
+			CalendarService calendarService,
+			DoctorService doctorService,
+			RepositoryAppointment appointmentRepository,
+			ModelMapper modelMapper) {
+		this.calendarService = calendarService;
+		this.doctorService = doctorService;
+		this.appointmentRepository = appointmentRepository;
+		this.modelMapper = modelMapper;
+	}
+
+	public List<AppointmentResponseDTO> getAllAppointments(UUID doctorUUID, UUID calendarUUID) {
```

Comment:
`getDoctorCalendarAppointments` ? Or only `getAppointments`, parameters should speak for themselves but `all` may be confusing

Reply:
Right now I had changed all these names, we will need these names with full info about what they do, swager2 will be more obvious for Wojtek